net/Socket.hpp

```diff
@@ -395,9 +466,17 @@ class Socket
 #endif
     }
 
-private:
     std::string _clientAddress;
+    const Type _type;
+    unsigned int _clientPort;
     const int _fd;
+    /// True if this socket is open.
+    std::atomic_bool _open; // FIXME: cache sync (atomic) required?
```

Comment:
We already have `_closed`, do we really need `_open` ?

Reply:
Moved StreamSocket::_closed -> Socket::_open, as described in commit message (besides other properties).
This move helps Socket::checkForcedRemoval() and SocketPoll::poll() has required properties provided.

Granted, SocketPoll::poll() only process StreamSocket objects ..

The isClosed() or isOpen() as well as setClosed() utilization now finds its endpoint in the Socket base class,
where Socket::shutdown() also ensures setClosed().